CTFd/api/v1/challenges.py

```diff
@@ -123,53 +124,49 @@ def get(self, query_args):
 
         # This can return None (unauth) if visibility is set to public
         user = get_current_user()
+        # Admins get a shortcut to see all challenges despite pre-requisites
+        admin_view = is_admin() and request.args.get("view") == "admin"
 
-        # Admins can request to see everything
-        if is_admin() and request.args.get("view") == "admin":
-            challenges = (
-                Challenges.query.filter_by(**query_args)
-                .filter(*filters)
```

Comment:
I've accidentally dropped this from the challenge query in its new location. Luckily the linter caught me. I'll re-add it